content/intro-to-storybook/vue/fr/contribute.md

```diff
@@ -0,0 +1,12 @@
+---
+title: 'Contribuer'
+description: 'Aider à partager Storybook dans le monde'
+---
+
+Les contributions à Learn Storybook sont encouragées ! S'il s'agit de petites corrections comme de la grammaire our de la ponctuation, ouvrez une pull request. S'il s'agit d'importants changements, [ajouter une issue](https://github.com/chromaui/learnstorybook.com/issues) pour échanger.
```

Comment:
our not exist in french :)
It must be 'ou'

Reply:
Thanks for all yours comments, i will change those next week !

Reply:
it's changed :)